### PR TITLE
Add option to show current temperature on thermostat card

### PR DESCRIPTION
**Pull request by piitaya**
## Proposed change

Add option to thermostat card to use current temperature instead of target temperature as main info.

CleanShot 2023-12-18 at 11 33 36

CleanShot 2023-12-18 at 11 35 12

PR for humidifier will follow.

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

**Comment by bramkragten** [comment]
Should the thermostat icon also be shown on a dual setpoint (if there is space)?

File: src/panels/lovelace/cards/types.ts [hunk 1]
```diff
@@ -455,6 +455,7 @@ export interface ThermostatCardConfig extends LovelaceCardConfig {
   entity: string;
   theme?: string;
   name?: string;
+  use_current_as_primary?: boolean;
```
[the remark this answers is not in the document]
Comment (piitaya) [reply src/panels/lovelace/cards/types.ts:458]: I wasn't sure for the name. I agree with your suggestion.

File: src/state-control/climate/ha-state-control-climate-temperature.ts [hunk 2]
```diff
@@ -163,36 +175,13 @@ export class HaStateControlClimateTemperature extends LitElement {
...
-    const actionLabel = this.hass.formatEntityAttributeValue(
-      this.stateObj,
-      "hvac_action"
-    );
-
     return html`
       <p class="label">
-        ${action && action !== "off" && action !== "idle"
-          ? actionLabel
-          : this.hass.localize("ui.card.climate.target")}
```
Comment (bramkragten) [inline src/state-control/climate/ha-state-control-climate-temperature.ts:195]: Lokalize is no longer used? Can be removed?

**Review by bramkragten — APPROVED** [review]

**Comment by andywwright** [comment]
Now all that is left is to make the old behavior default and then we could forget all this nonsense as if it has never happened

**Pull request merged**